Core/Core/Logging/Analytics.cs

```diff
@@ -221,17 +221,20 @@ public enum Events
 
         if (customProperties != null)
         {
-          properties = properties.Concat(customProperties).ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
+          foreach (KeyValuePair<string, object> customProp in customProperties)
+          {
+            properties[customProp.Key] = customProp.Value;
+          }
         }
 
         string json = JsonConvert.SerializeObject(new { @event = eventName.ToString(), properties });
 
         var query = new StreamContent(new MemoryStream(Encoding.UTF8.GetBytes("data=" + HttpUtility.UrlEncode(json))));
 
-        using HttpClient client = Http.GetHttpProxyClient();
+        using HttpClient client = new();
         client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("text/plain"));
         query.Headers.ContentType = new MediaTypeHeaderValue("application/json");
-        client.PostAsync(MIXPANEL_SERVER + "/track?ip=1", query);
```

Comment:
There's a similar call to PostAsync in the AddConnectorToProfile.

I imagine this one will need the same change.